.gofmt.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+fmt_errors=$(gofmt -d .)
```

Comment:
fair enough. i'll change it to be like the other scripts for now, we can revisit later if we need to.